### DISCUSSION TITLE
Why can't i get property nbt.value.value from a item object?

### DISCUSSION THREAD

**Discussion by Daniel-Glz** [PrismarineJS/mineflayer#2312] [Q&A]
I was trying to get the type of a splash potion through **nbt** property of a item, but i can't get it. When i try to get it returns me **undefined**, Why is happening this?

My code:
```
const mineflayer = require('mineflayer');

const options = {
    host:"localhost",
    username:"ImABot",
}

const bot = mineflayer.createBot(options);


bot.on('chat', () => {
    let inventoryArr = bot.inventory;
    let potion = inventoryArr.findInventoryItem(438).nbt.value;
    console.log(`item.nbt.value: `);
    console.log(potion);
    console.log(`\n\nitem.nbt.value.value:\n ${potion.value}`);
});

```

When i run my code, potion returns me this object `{ Potion: { type: 'string', value: 'minecraft:strong_healing' } }`
and i try to get its **value property** but it returns me undefined

**Comment by u9g** [comment]
.Potion.value not just .value
